state/errors.go

```diff
@@ -142,3 +142,5 @@ func IsUnknownBlockError(err error) bool {
 	var e UnknownBlockError
 	return errors.As(err, &e)
 }
+
+var ErrNoVersionBeacon = errors.New("no version beacon exists")
```

Comment:
It might be nice to include the block height for which a version beacon was requested. In case there are version beacons, just not "low" enough.

Reply:
This error is intended for when snapshot doesn't have the version beacon available, so there is no given height per se.